central/role/resources/list.go

```diff
@@ -24,56 +24,92 @@ import (
 //
 // KEEP THE FOLLOWING LIST SORTED IN LEXICOGRAPHIC ORDER (case-sensitive).
 var (
-	APIToken                         = newResourceMetadata("APIToken", permissions.GlobalScope)
-	Alert                            = newResourceMetadata("Alert", permissions.NamespaceScope)
-	AllComments                      = newResourceMetadata("AllComments", permissions.GlobalScope)
-	AuthPlugin                       = newResourceMetadata("AuthPlugin", permissions.GlobalScope)
-	AuthProvider                     = newResourceMetadata("AuthProvider", permissions.GlobalScope)
-	BackupPlugins                    = newResourceMetadata("BackupPlugins", permissions.GlobalScope)
-	CVE                              = newResourceMetadata("CVE", permissions.NamespaceScope)
-	Cluster                          = newResourceMetadata("Cluster", permissions.ClusterScope)
-	Compliance                       = newResourceMetadata("Compliance", permissions.ClusterScope)
-	ComplianceRunSchedule            = newResourceMetadata("ComplianceRunSchedule", permissions.GlobalScope)
-	ComplianceRuns                   = newResourceMetadata("ComplianceRuns", permissions.ClusterScope)
-	Config                           = newResourceMetadata("Config", permissions.GlobalScope)
-	DebugLogs                        = newResourceMetadata("DebugLogs", permissions.GlobalScope)
-	Deployment                       = newResourceMetadata("Deployment", permissions.NamespaceScope)
-	Detection                        = newResourceMetadata("Detection", permissions.GlobalScope)
-	Group                            = newResourceMetadata("Group", permissions.GlobalScope)
-	Image                            = newResourceMetadata("Image", permissions.NamespaceScope)
-	ImageComponent                   = newResourceMetadata("ImageComponent", permissions.NamespaceScope)
-	ImageIntegration                 = newResourceMetadata("ImageIntegration", permissions.GlobalScope)
+	// Access is the new resource grouping all access related resources.
+	Access = newResourceMetadata("Access", permissions.GlobalScope)
+	// Administration is the new resource grouping all administration-like resources.
+	Administration = newResourceMetadata("Administration", permissions.GlobalScope)
+	Alert          = newResourceMetadata("Alert", permissions.NamespaceScope)
+	CVE            = newResourceMetadata("CVE", permissions.NamespaceScope)
```

Comment:
Fun fact, this actually doesn't give you access to Vuln Management page. No idea what it actually does. (cc @md2119 )

Reply:
CVE sac is unused and should be removed. For components and vulns we use images, nodes, or clusters access. Same for ImageComponent, if it exists.